### PR TITLE
12174 mhr staff payment flow priority fee

### PR DESCRIPTION
**Pull request by garyfan1**
*Issue #:* /bcgov/entity#12544

*Description of changes:*
- implement priority fee
- added test for priority fee
- show error label and border when validation fails

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/views/ConfirmMHRSearch.vue [hunk 1]
```diff
@@ -355,4 +356,10 @@ export default class ConfirmDischarge extends Vue {
 
 <style lang="scss" module>
 @import '@/assets/styles/theme.scss';
+
+#staff-payment-dialog {
+  .title-label{
+    color: red !important;
+  }
+}
```
Comment (cameron-eyds) [inline ppr-ui/src/views/ConfirmMHRSearch.vue]: Is this required?

File: ppr-ui/src/views/ConfirmMHRSearch.vue [hunk 2]
```diff
@@ -52,13 +52,14 @@
               </v-col>
             </v-row>
 
-            <v-card flat class="mt-6 pa-6">
+            <v-card flat class="mt-6 pa-6" :class="staffPaymentValid? '' :'border-error-left'">
```
Comment (cameron-eyds) [inline ppr-ui/src/views/ConfirmMHRSearch.vue]: I don't mind the ternary in the template but i would add the spaces you would use if in a script for readability. 
ie `staffPaymentValid ?  ''  :  'border-error-left'`

**Review by cameron-eyds — COMMENTED** [review]
Looks good just some minor comments

File: ppr-ui/src/views/ConfirmMHRSearch.vue (hunk 1 above)
Comment (garyfan1) [reply ppr-ui/src/views/ConfirmMHRSearch.vue]: oops, no, not at all

File: ppr-ui/src/views/ConfirmMHRSearch.vue (hunk 2 above)
Comment (garyfan1) [reply ppr-ui/src/views/ConfirmMHRSearch.vue]: updated!

**Review by cameron-eyds — APPROVED** [review]
Nice!

**Pull request merged**